src/main/resources/interface-specification.yml

```diff
@@ -7193,6 +7265,8 @@ components:
           type: string
         producer:
           $ref: '#/components/schemas/CompactOrganization'
+        descriptor:
```

Comment:
Which is the meaning of this descriptor? Is that one related to purpose? In this case, I agree with @galales. This model must be agnostic to purpose because it describes a compact view of eservice.
Adding a new dedicated model looks like the best choice even if it adds more verbosity.